_pages/security/dynamic-scanning.md

```diff
@@ -53,10 +53,18 @@ Authentication can be handled in several ways; the following links provide guida
 
 The Attack Script runs the Active Scan and Spider functions. If your page is a javascript-driven single page application, you will want to run the Ajax Spider in addition to the standard spider. The Ajax Spider will instantiate a browser window and spider the application visually. This is significantly slower than the standard spider. Read more about the Ajax Spider [here](https://github.com/zaproxy/zap-core-help/wiki/HelpAddonsSpiderAjaxConcepts).
 
+#### Fuzzing
+
+["Fuzzing"](https://en.wikipedia.org/wiki/Fuzz_testing) refers to feeding a large amount of random (and/or potentially malicious) data to an application with the intention of finding vulnerabilities related to poor error handling or incomplete input validation. 
```

Comment:
I'd add something to the effect of "basically, use this anywhere you rely on query param or have a form".